internal/incident/incident.go

```diff
@@ -462,10 +462,18 @@ func (i *Incident) notifyContacts(ctx context.Context, tx *sqlx.Tx, ev *event.Ev
 			return err
 		}
 
-		for chType := range channels {
+		for chId := range channels {
+			chConf := i.runtimeConfig.Channels[chId]
+			if chConf == nil {
+				i.logger.Errorw("Could not find config for channel", zap.Int64("channel_id", chId))
+				continue
+			}
+
+			chType := chConf.Type
+
 			i.logger.Infof("Notify contact %q via %q", contact.FullName, chType)
```

Comment:
This should now also include the ID, or probably even better, the name from the channel config. One side effect of this change is now that you can for example configure multiple RocketChat channels that correspond to different servers. And if you do, it's good to know which one was referenced here.